tests/device_fixtures.py

```diff
@@ -407,22 +408,28 @@ async def _discover_update_and_close(ip, username, password) -> Device:
     return await _update_and_close(d)
 
 
-async def get_device_for_fixture(fixture_data: FixtureInfo) -> Device:
+async def get_device_for_fixture(
+    fixture_data: FixtureInfo, *, verbatim=False, update=True
```

Comment:
```suggestion
    fixture_data: FixtureInfo, *, verbatim=False, force_update=True
```
`force_update`? `update_after_init`? Or something else to make clear why this is set or not?

Reply:
Renamed `update_after_init`